src/foam/nanos/auth/User.js

```diff
@@ -448,14 +450,21 @@ foam.CLASS({
       name: 'disabledTopics',
       documentation: 'Disables types for notifications.',
       createVisibility: 'HIDDEN',
-      section: 'administrative'
+      section: 'administrative',
+      javaPostSet: `
```

Comment:
Better to create in the factory: for disabledTopicSet() and just clear that property in the post set here. That prevents you from having to generate the set if not used. I would rather just use a for loop to loop over val rather than create an intermediate Array which then needs to be garbate collected.